docs/features.md

```diff
@@ -0,0 +1,55 @@
+# pymwp Features
+
+## Supported C-Language Features
+
+This section provides a summary of supported C language features pymwp can 
+analyze. It also lists language features that are in the process of being 
+implemented and for which implementation is planned.
+
+!!! info "Note on C file parsing"
+    pymwp uses pycparser to process the C input file. Any limitation of 
+    pycparser is also a limitation in pymwp. From
+    [pycparser website](https://github.com/eliben/pycparser):
+    "pycparser aims to support the full C99 language (according to the 
+    standard ISO/IEC 9899). Some features from C11 are also supported."
+  
+**Legend**
+
+- ✅ &nbsp; **ready** - fully implemented and ready to use
+- 🟧 &nbsp; **in progress** - implementation is in progress but not ready
```

Comment:
We could mark "Header Files Inclusion" as "in progress": as soon as we have compositionality, we should be able to issue libraries a certificate and get the coefficients we need from them to use in the analysis of the main methods.